queue/ready-queue.js

```diff
@@ -111,24 +139,28 @@ module.exports = function (RED) {
 				statusOutput();
 			}
 			setStatusTimer() ;
+			setFileSizeTimer();

```

Comment:
If its called on `add` to start polling the file size, and is called again to stop on `empty` event, is it necessary to call setFileSizeTimer() here?

Reply:
The github diff here is not ideal. It looks like this method is called on `next` but it is called on `start` (row 137), same as `setStatusTimer()` (row 141). I think it is important to call `setStatusTimer()` and  `setFileSizeTimer()` on the `start` event because both their implementations are using something like:

```
if(!isConnected || (isConnected && !queue.isEmpty()))
```

In other words the outcome of these functions (start or stop the timer) is dependent on the `start` or `stop` status of the queue.